app/components/lg-lion-search.js

```diff
@@ -54,9 +61,31 @@ export default Ember.Component.extend({
       params['name'] = selectedName;
     }
 
+    if (selectedDob && !Ember.isBlank(selectedDob)) {
```

Comment:
@redyaffle  This doesn't seem to be working, even if I choose something in the age drop down, selectedDob is still null.